scripts/bundle.ts

```diff
@@ -88,10 +93,11 @@ async function bundle() {
         ],
         // mark all packages that are not *this* package as external so they don't get included in the bundle
         // include tslib in the bundles since only __decorate is really used by multiple packages (we can figure out a way to deduplicate that later on if need be)
-        external: project.packages
-          .filter((p) => p.name !== pkg.name)
-          .map((p) => p.scopedName)
-          .concat(['cross-fetch']),
+        external: getKeys(pkg.name),
+        // external: project.packages
+        //   .filter((p) => p.name !== pkg.name)
+        //   .map((p) => p.scopedName)
+        //   .concat(['cross-fetch']),
```

Comment:
Are these comments necessary?

Reply:
We can just leave it here for now as we cannot make sure the changed bundle script can work will util we do actual `publish`. My suggestion is to delete it next pr if everything works well.

Reply:
sure, let's do that.